README.md

```diff
@@ -6,6 +6,42 @@
 
 The Wrapper Upgrade Gradle Plugin creates tasks to upgrade the Gradle Wrapper for target projects hosted on GitHub.
 
+## Prerequisites
+
+To run the upgrade tasks, you'll need:
+
+* **Java 8 or later.**
+* **Git CLI.**  The plugin uses the Git CLI to commit, create branches, and push changes.
```

Comment:
Maybe add a link to the CLI, and I'd call it GitHub CLI.

Reply:
It's not the github cli, it's just "git" 🙃

Reply:
Then I'd just call it Git.

Reply:
done